packages/auth/src/Domain/Feature/Feature.ts

```diff
@@ -0,0 +1,67 @@
+import { ComponentArea, ContentType } from '@standardnotes/snjs';
+
+export enum FeatureName {
```

Comment:
Hmm maybe this should be `PermissionName` if they are the same ?

Reply:
I'm thinking we'll get rid of the Permission and Role types on SNJS once clients no longer deal with permissions?

Reply:
yeah, the server uses the `auth` package